src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/resource/EndpointSecurityCheck.kt

```diff
@@ -34,11 +36,13 @@ class EndpointSecurityCheck {
     }
   }
 
+  private val excluded = setOf(HmppsQueueResource::class.java, HmppsReactiveQueueResource::class.java)
+
   private fun getAllUnprotectedControllers() = ClassPathScanningCandidateComponentProvider(false)
     .also { it.addIncludeFilter(AnnotationTypeFilter(RestController::class.java)) }
     .findCandidateComponents("uk.gov.justice")
     .map { Class.forName(it.beanClassName) }
-    .filter { !it.isProtectedByAnnotation() }
+    .filter { !it.isProtectedByAnnotation() && !excluded.contains(it) }
```

Comment:
Not sure these should be excluded as they should be behind the RESETTLEMENT_PASSPORT_EDIT role

Reply:
They are endpoints for managing dead letter queues exposed automatically by the hmpps-sqs-spring-boot-starter. Did they have it before and have lost it somehow or are they new endpoints (which was my assumption)

Reply:
Either way - I can't inject the annotation on them, I'd have to add the routes specifically to the `ResourceServerConfiguration` file so the test would have to be separate

Reply:
These are existing endpoints that were set up so we can intergation with SQS in the future. You can set the role in the application.yml: https://github.com/ministryofjustice/hmpps-resettlement-passport-api/blob/main/src/main/resources/application.yml#L118 so not sure why that has stopped working...

Reply:
👍